core/templates/components/certificate-assessment-offering-helper/certificate-offering-details.component.css

```
@@ -0,0 +1,236 @@
+.oppia-certificate-offering-details {
+  color: #333;
+  font-family: 'Roboto', sans-serif;
```

Stylelint (17.12.0)

[error] Expected no quotes around "Roboto" (font-family-name-quotes)

(font-family-name-quotes)